app/javascript/dashboard/routes/dashboard/contacts/components/Header.vue

```diff
@@ -155,4 +181,17 @@ export default {
     visibility: visible;
   }
 }
+.filters__button-wrap {
+  position: relative;
+
+  .filters__applied-indicator {
```

Comment:
Use variables

Reply:
Fixed in https://github.com/chatwoot/chatwoot/pull/3471/commits/ec63d94adad3ec1d868806c97611996d7bba4e9a